opentelemetry/src/lib.rs

```diff
@@ -299,3 +299,9 @@ pub mod time {
         SystemTime::UNIX_EPOCH + std::time::Duration::from_millis(js_sys::Date::now() as u64)
     }
 }
+
+#[doc(hidden)]
+pub mod _private {
+    #[cfg(feature = "internal-logs")]
+    pub use tracing; // re-export
```

Comment:
should this expose entire tracing module or selected components we care about?

Reply:
I exported the macros we actually use now. at the end we can always change it. it is not meant to be used directly by someone else than us.